ui/src/app/components/prepare-migration/prepare-migration.component.html

```diff
@@ -118,7 +118,7 @@ <h3>Target details:</h3>
                             [disabled]="isMigrationInProgress || !isTargetDetailSet">
                             Configure
                             <mat-icon>edit</mat-icon>
-                            <mat-icon *ngIf="isDataflowConfigurationSet" class="success"
+                            <mat-icon class="success"
```

Comment:
If we remove this if will it not show the icon always?

Reply:
Yes, that's the experience we want (Want the user to think Dataflow is already configured unless they know what they are doing by tweaking the config)